cpu/sam0_common/periph/spi.c

```diff
@@ -142,10 +147,6 @@ int spi_acquire(spi_t bus, spi_cs_t cs, spi_mode_t mode, spi_clk_t clk)
 
 void spi_release(spi_t bus)
 {
-    /* disable device and put it back to sleep */
-    dev(bus)->CTRLA.reg &= ~(SERCOM_SPI_CTRLA_ENABLE);
-    while (dev(bus)->SYNCBUSY.reg & SERCOM_SPI_SYNCBUSY_ENABLE) {}
-    poweroff(bus);
```

Comment:
I don't think I tested that explicitly. I'll be back in the lab in about 4 hours and I'll test that then.